docs/Project.toml

```diff
@@ -0,0 +1,6 @@
+[deps]
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
```

Comment:
```suggestion
```
This should only be in there if a manifest is committed that tells it that the path is relative to this directory, otherwise documentation will always try to use the release version of FFTW.